buildSrc/src/main/kotlin/LibsConfig.kt

```diff
@@ -87,6 +88,10 @@ fun Project.applyLibrariesConfiguration() {
         from({
             altConfigFiles("sources")
         })
+
+        // Yeet module-info's
+        exclude("module-info.java")
```

Comment:
These cause duplications in the sources jar, hence excluding _.java_ rather than _.class_